package_policies/M17.md

```diff
@@ -1,8 +1,10 @@
 **R8.** Each xSDK member package should have sufficient documentation to support use and further development.
 While it is difficult to formalize what constitutes sufficiency of documentation, the requirement will be reviewed by the
-xSDK community based on documentation elements that may include, for example, the following:
+xSDK community based on documentation elements and must include:
   * Instructions on installation and getting started;
   * Documentation on functionality, for example class- and member-level documentation (in object-oriented languages) or more generally API-level documentation;
+
+It is also recommended that the documentation include, for example, the following:
```

Comment:
I suggest to change this to
It is recommended that the documentation also include the following:

Reply:
Agreed